packages/opencensus-instrumentation-http2/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "noImplicitThis": false,
```

Comment:
Is it possible to remove this compiler option?

Reply:
As we use `original.apply(this, argumets)` a lot, removing this compiler option causes a lot of trouble in this plugin.

Reply:
TypeScript has a workaround for this -- anywhere `this` is needed in an ES5 function, you can use this syntax:

```ts
// `this` is now explicit
function (this: Http2Server, arg1: string, /*...*/) {
  this.connect();
}
```

Reply:
Great to know. Updated and removed the compiler option.